wrapper/package-lock.json

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
Can we drop this from the repo? It was previously removed and this seems to add it back. I think we use slightly different practices in this repo compared to some other js projects (I'm not very familiar with js repo management in practice).